crates/interfaces/Cargo.toml

```diff
@@ -54,4 +54,5 @@ secp256k1 = { workspace = true, features = [
 
 [features]
 test-utils = ["tokio-stream/sync", "secp256k1", "rand/std_rng"]
-cli = ["clap"]
\ No newline at end of file
+cli = ["clap"]
+optimism = ["reth-eth-wire/optimism"]
```

Comment:
I don't see the `"reth-eth-wire/optimism"` dependency being used. We should verify that it isn't used in `clabby/op-reth`.